### PR TITLE
[minigraph][LAG] Add LAG minigraph for Mellanox msn-2700 platform

### PR DESCRIPTION
**Pull request by hrachya-mellanox**
* new minigraph file for Mellanox platform
 * fix minigraph_facts.py to handle Mellanox msn-2700

 signed-off-by hrachya@mellanox.com

### CODE REVIEW

File: ansible/library/minigraph_facts.py [hunk 1]
```diff
@@ -349,7 +349,7 @@ def parse_xml(filename, hostname):
             hwsku = child.text
 
     # port_alias_map maps ngs port name to sonic port name
-    if hwsku == "Force10-S6000":
+    if hwsku == "Force10-S6000" or hwsku == "ACS-MSN2700":
```
Comment (stcheng) [inline ansible/library/minigraph_facts.py]: I notice that switch2.xml file is also describing ACS-MSN2700. So it is better to replace the current minigraph interfaces' name to Ethernet*. So we don't need to have such logic here.

File: ansible/minigraph/switchlag.xml [hunk 2]
```diff
@@ -0,0 +1,916 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```
Comment (stcheng) [inline ansible/minigraph/switchlag.xml]: Use the name switch5.xml to align with the current names.
The topology description could be commented in the inventory file.

File: ansible/minigraph/switchlag.xml [hunk 3]
```diff
@@ -0,0 +1,916 @@
...
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>switchlag</EndDevice>
+        <EndPort>fortyGigE0/124</EndPort>
+        <StartDevice>ARISTA16T0</StartDevice>
+        <StartPort>Ethernet1</StartPort>
+      </DeviceLinkBase>
+    </DeviceInterfaceLinks>
+  </PngDec>
+  <Hostname>switchlag</Hostname>
```
Comment (stcheng) [inline ansible/minigraph/switchlag.xml]: Name it switch5

File: ansible/minigraph/switchlag.xml [hunk 4]
```diff
@@ -0,0 +1,916 @@
...
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>switchlag</EndDevice>
+        <EndPort>fortyGigE0/120</EndPort>
+        <StartDevice>ARISTA15T0</StartDevice>
+        <StartPort>Ethernet1</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>switchlag</EndDevice>
+        <EndPort>fortyGigE0/124</EndPort>
```
Comment (stcheng) [inline ansible/minigraph/switchlag.xml]: Replace the interface name with Ethernet124 etc.

**Comment by lguohan** [comment]
also, can you add this switch5 to the inventory file?

**Pull request merged**